src/homekit.cpp

```diff
@@ -23,13 +23,21 @@ void target_door_state_set(const homekit_value_t new_value);
 homekit_value_t obstruction_detected_get();
 homekit_value_t active_state_get();
 
+// Make device_name available
+extern "C" char device_name[19];
```

Comment:
consider making the numbers 19 and 7 into `const usize` variables or `#define`s, since they appear in multiple places.